centos/centos-7.x/centos-7.6-hpc/install_mellanoxofed.sh

```diff
@@ -3,6 +3,7 @@ set -ex
 
 MLNX_OFED_DOWNLOAD_URL=https://azhpcstor.blob.core.windows.net/azhpc-images-store/MLNX_OFED_LINUX-5.4-1.0.3.0-rhel7.6-x86_64.tgz
 TARBALL=$(basename ${MLNX_OFED_DOWNLOAD_URL})
+echo "MOFED="$(echo $TARBALL | awk -F- '{print $2 "-" $3}') | sudo tee -a $IMAGE_DIR/component_versions.txt
```

Comment:
Consider a case where the file name format changes. Someone will update the file name on the line above but might forget to update this. And as far as my understanding goes that line is not gonna throw an error.
Here two lines require changes.

Think about something like this,
```
version =- "5.4-1.0.3.0"
echo $version | sudo tee -a $IMAGE_DIR/component_versions.txt
tarball = "MLNX_OFED_LINUX-$version-rhel7.6-x86_64.tgz"
MLNX_OFED_DOWNLOAD_URL=https://azhpcstor.blob.core.windows.net/azhpc-images-store/$tarball
```
Here, you just have to update one line.

Reply:
```
sudo tee -a .......
```
This is at multiple places. If you want you can make it a function in common_dir. Here it could be something like,
```
WRITE_VERSION $version
````